AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -81,16 +81,20 @@ struct ConfirmCreatingSessionView: View {
                 VStack(alignment: .leading, spacing: 15) {
                     if sessionContext.sessionType == .fixed {
                         descriptionTextFixed
-                    } else {
+                    } else if sessionContext.locationless != true {
                         descriptionTextMobile
+                    } else {
+                        defaultDescriptionText
                     }
                 }
                 .font(Fonts.muliHeading2)
                 .foregroundColor(Color.aircastingGray)
                 .lineSpacing(9.0)
                 ZStack {
                     if sessionContext.sessionType == .mobile {
-                        GoogleMapView(pathPoints: [], isMyLocationEnabled: true, placePickerDismissed: Binding.constant(false), isUserInteracting: Binding.constant(true))
+                        if sessionContext.locationless != true {
```

Comment:
same, a bit weird this bool expression is. Can also be added as an `&&` to the `if` above

Reply:
Not really I think, cause then locationfull (🙃) mobile sessions will fall into the else

Reply:
That's true. I'm changing locationless to Bool (from Bool?) and giving it default value of false. This will simplify all those != true etc.